tabulator/parsers/datapackage.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import datapackage
+import six
+
+from ..parser import Parser
+
+
+# Module API
+
+class DataPackageParser(Parser):
+    """Parser to extract data from Tabular Data Packages.
+
+    See: http://specs.frictionlessdata.io/
+    """
+
+    # Public
+
+    options = [
+        'resource',
+    ]
+
+    def __init__(self, loader, resource=0):
+        self.__resource = resource
+        self.__extended_rows = None
+        self.__force_parse = None
+        self.__datapackage = None
+        self.__resource_iter = None
+
+    @property
+    def closed(self):
+        return self.__resource_iter is None
+
+    def open(self, source, encoding=None, force_parse=False):
+        self.close()
+        self.__force_parse = force_parse
+        self.__datapackage = datapackage.DataPackage(source)
+        self.reset()
+
+    def close(self):
+        if not self.closed:
+            self.__datapackage = None
+            self.__resource_iter = None
+            self.__extended_rows = None
+
+    def reset(self):
+        if isinstance(self.__resource, six.string_types):
+            print(self.__datapackage.resources)
+            named_resource = next(iter(filter(
```

Comment:
We will be able to rebase on `data_package.getResource(name)` after `datapackage-v1` release.